components/RuiLayout/tests/RuiLayout.test.ts

```diff
@@ -7,7 +7,7 @@
 */
 
 import '../src';
-import { TestUtils } from '../../../TestUtils';
+import TestUtils from '../../../TestUtils';
```

Comment:
You might find linter will complain about this. If so change back but remove the spaces eg `{TestUtils}` is good.

Reply:
@jennasalau weird, I changed it to default export because the linter was complaining haha